src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -217,6 +217,8 @@ public BulkInsertOperation(string database, IDocumentStore store, CancellationTo
 
             _token = token;
             _conventions = store.Conventions;
+            if (string.IsNullOrWhiteSpace(database))
+                throw new ArgumentNullException(nameof(database), "Document store database name cannot be empty");
```

Comment:
Please make it more user friendly, similar to what we have here: 
https://github.com/ravendb/ravendb/blob/f2fad77b13ce1f94a80b84c549fd615ad70db1c5/src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs#L1119